.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,55 +6,31 @@ labels: 'bug'
 assignees: ''
 
 ---
+## 🐛 Bug Report - Please complete all 6 steps! 
 
-## 🐛 Bug Report
+### 1. Environment: (Run `expo diagnostics` and copy the output)
 
-### Environment
+Replace this text with log output.
 
-<!-- In your terminal run `expo diagnostics` and paste the output here. -->
+### 2. Targets: (iOS, Android, Web, Client, Standalone, ExpoKit)
```

Comment:
i think this one would be a bit confusing for people. we might want to have platform (ios/android/web) and then environment (client/bare/standalone/expokit)